### PR TITLE
[All-inkl] Remove right-wildcard and simplify ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**
https://github.com/EFForg/https-everywhere/issues/11203

### CODE REVIEW

File: src/chrome/content/rules/All-Inkl.xml [hunk 1]
```diff
@@ -1,23 +1,52 @@
+<!--
+	No working URL known:
+		all-inkl.es
+		www.all-inkl.es
+
+	Invalid certificate:
+		(www.)?all-inkl.ch
+		(www.)?all-inkl.de
+		(www.)?all-inkl.it
+		(www.)?all-inkl.uk
```
Comment (J0WI) [inline src/chrome/content/rules/All-Inkl.xml]: There are much more like this

Comment (Bisaloo) [inline src/chrome/content/rules/All-Inkl.xml]: you mean more TLDs?

Comment (J0WI) [inline src/chrome/content/rules/All-Inkl.xml]: yes

Comment (Bisaloo) [inline src/chrome/content/rules/All-Inkl.xml]: @J0WI, done. I checked every single ccTLD. They don't own all of them so the previous rewrite wasn't really safe/correct.

**Pull request merged**